app/src/main/java/com/example/keniphhomemaintenance/MainActivity.kt

```diff
@@ -3,36 +3,14 @@ package com.example.keniphhomemaintenance
 import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
-import androidx.compose.material.MaterialTheme
-import androidx.compose.material.Surface
-import androidx.compose.material.Text
-import androidx.compose.runtime.Composable
-import androidx.compose.ui.tooling.preview.Preview
-import com.example.keniphhomemaintenance.ui.theme.KeniphHomeMaintenanceTheme
+import androidx.compose.ui.ExperimentalComposeUiApi
 
 class MainActivity : ComponentActivity() {
+    @ExperimentalComposeUiApi
```

Comment:
Annotation is for Navigation()

Reply:
We should be able to get rid of this annotation. See my comment in DwellingScreenView.kt

Reply:
All gone